README.md

```diff
@@ -1,41 +1,39 @@
 1. Encourage more Kagglers to start working on this competition.
 1. Deliver open source solution with no strings attached. Code is available on our [GitHub repository :computer:](https://github.com/neptune-ml/open-solution-salt-detection). This solution should establish solid benchmark, as well as provide good base for your custom ideas and experiments. We care about clean code :smiley:
-1. We are opening our experiments as well: everybody can have **live preview** on our experiments, parameters, code, etc. Check: [TGS Salt Identification Challenge :chart_with_upwards_trend:](https://app.neptune.ml/neptune-ml/Salt-Detection) and screens below.
+1. We are opening our experiments as well: everybody can have **live preview** on our experiments, parameters, code, etc. Check: [TGS Salt Identification Challenge :chart_with_upwards_trend:](https://app.neptune.ml/neptune-ml/Salt-Detection) or screen below.
 
-| Train and validation monitor :bar_chart: |
-|:---|:---|
-|[![training monitor](https://gist.githubusercontent.com/jakubczakon/cac72983726a970690ba7c33708e100b/raw/b45dd02b6643a3805db42ab51a62293a2940c0be/neptune_salt.png)|
+|Train and validation monitor :bar_chart:|
+|:---:|
+|[![training monitor](https://gist.githubusercontent.com/jakubczakon/cac72983726a970690ba7c33708e100b/raw/b45dd02b6643a3805db42ab51a62293a2940c0be/neptune_salt.png)](https://app.neptune.ml/-/dashboard/experiment/3dfce6cf-3031-4e9a-b95c-1ac8b5bb0026)|
 
 ## Disclaimer
 In this open source solution you will find references to the [neptune.ml](https://neptune.ml). It is free platform for community Users, which we use daily to keep track of our experiments. Please note that using neptune.ml is not necessary to proceed with this solution. You may run it as plain Python script :snake:.
 
 # How to start?
 ## Learn about our solutions
-1. Check [Kaggle forum](TODO) and participate in the discussions.
-1. Check our [Wiki pages :house_with_garden:](https://github.com/neptune-ml/open-solution-salt-detection/wiki), where we document our work. See solutions below:
+1. Check [Kaggle forum](https://www.kaggle.com/c/tgs-salt-identification-challenge/discussion/61949) and participate in the discussions.
+1. See solutions below:
 
-| link to code | name | CV | LB | link to description |
-|:---:|:---:|:---:|:---:|:---:|
-|[solution 1](https://github.com/neptune-ml/open-solution-salt-detection/tree/solution-1)|*chestnut* :chestnut:|0.413|0.745|[Vanilla Unet](https://github.com/neptune-ml/open-solution-salt-detection/wiki/TODO)|
-|solution 2|*four leaf clover* :four_leaf_clover:|0.794|0.798|[Deeper unet with smart augmentations](https://github.com/neptune-ml/open-solution-salt-detection/wiki/TODO)||
+| link to code | CV | LB |
+|:---:|:---:|:---:|
+|[solution 1](https://github.com/neptune-ml/open-solution-salt-detection/tree/solution-1)|0.413|0.745|
```

Comment:
@jakubczakon are you sure CV is 0.413 ?

Reply:
Yup we had an error in metric and that was what it was reporting :)